file-format-spec.ts

```diff
@@ -0,0 +1,58 @@
+// This file contains types which specify the speedscope file format.
+
+export interface SerializedFrame {
+  name: string
+  file?: string
+  line?: number
+  col?: number
+}
+
+export interface SerializedNode {
```

Comment:
would it be possible to add optional start + duration numbers to nodes so we can import non-sampling data sources?